[NT-1016] Update app strings

ifbarrera opened the pull request:
# 📲 What

A big update of a bunch of translation strings. Also adds screenshot tests for `LandingPageViewController`, which were missing.

# 🤔 Why

Keeping our translations up to date ✨ 

# 🛠 How

- ran `make strings`
- updated and verified any breaking screenshots
- replaced any instances of `localizedString()` with the correct `Strings` functions
- added `LandingPageViewControllerTests` for screenshots of the Landing Page screen

# ✅ Acceptance criteria

- [x] skim all the screenshots 😬

nativeksr commented:
<!--
  0 Errors
  1 Warning: Big PR
  0 Messages
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

ifbarrera commented:
> Seems fine, the background looks weird on this device but I'm guessing it's just aspect ratio. Wonder if we can't just let it fill and see how it looks? Might be better for the 4.7 inch device. Also, is the button meant to be edge-to-edge like that?
> 
> <img alt="image" width="50%" src="https://user-images.githubusercontent.com/3735375/78956545-77654f80-7a97-11ea-8c4a-17eb3fb9cb2b.png">

Good eye, the `contentMode` was previously set to `.scaleToFill`. I've updated it to `.scaleAspectFill` and re-recorded the snapshots. It's not meant to be edge-to-edge, but I think because the scroll view is pinned to the margins, and there aren't any margins on the parent view controller, it's showing up as edge-to-edge. I've updated the parent view's margins and re-recorded.
